### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ExtractDacpac_ExtractDacpacSettings_Screenreader: The controls present beside source server and source database are not announced correctly.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6847]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAC 10.14.6
Tools: Voiceover
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select second radio button.
9. Navigate till next button & hit enter.
10. Start screen reader as voiceover and navigate to source server combo-box.

**Actual:**
**Issue1:**
Navigate to the source server combo-box , screen reader announces it as "pop-up button collapse". Similar issue for source database combo-box

**Issue2:**
Navigate to the source server combo-box and expand it  , screen reader announces it as "menu" along with the items as menu item 1.Similar issue for source database combo-box

**Same issue is observed for the following page:**
1.Import bacpac setting:
Follow the exact above steps but select the third radio button
Issue observed for  "Target server".

**Expected:**
**Issue1:**
Navigate to the source server combo-box , screen reader should  announces it as "combo-box collapsed".

**Issue2:**
Navigate to the source server combo-box and expand it  , screen reader announces it as "list with 1 item"

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not receive the right information about the controls displayed on screen as well as the action to be performed on them.

**MAS Reference:**
MAS1.3.1:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[#6847A11y_AzureDataStudio(Schema,Dacpac,Backup)_ExtractDacpac_ExtractDacpacSettings_Screenreader.mp4.pptx](https://github.com/microsoft/azuredatastudio/files/3524031/6847A11y_AzureDataStudio.Schema.Dacpac.Backup._ExtractDacpac_ExtractDacpacSettings_Screenreader.mp4.pptx)

**Comment by udeeshagautam** [comment]
Hi @fsteffi , I am not able to repro this. I am getting the combobox <text> collapsed from NVDA. can you please help with the repro of this issue

![image](https://user-images.githubusercontent.com/46980425/63462531-fd9d2a80-c40f-11e9-835a-1ba521c19c48.png)

**Comment by udeeshagautam** [comment]
Is this a voiceover only issue? Adding to reader specific issues

**Comment by fsteffi** [comment]
Hi @udeeshagautam this bug is specific to voiceover.

**Comment by kisantia** [comment]
Confirmed that this is expected behavior on Mac.
![image](https://user-images.githubusercontent.com/31145923/64371378-3bd04780-cfd5-11e9-9854-2784f6a5d19b.png)
